src/js/components/FileInput/__tests__/FileInput-test.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import 'jest-styled-components';
+import { render } from '@testing-library/react';
+
+import { Grommet } from '../../Grommet';
+import { FileInput } from '..';
+
+describe('FileInput', () => {
```

Comment:
Can we add a test of the component used within Form, or assertion of the actual action of uploading a file or multiple files?

Reply:
Hmm... I can investigate. So much of this is handled by the `<input type="file" />` element. I think we'd have to find a way to mock that.